webpages/popup/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Could this cause some issues on desktop? 🤔 The size of the popup has frequently created problems.
Maybe to be on the safe side we should add this dynamically with JavaScript if the user agent matches mobile Chromium or mobile Firefox?

Reply:
I haven't noticed any problems so far and Vue 3 also adds it but it's better to be safe I guess.

Reply:
Unless it's handled differently in extension popups for some reason, the viewport tag has no effect on desktop.

Reply:
Then let's assume this didn't break anything unless we've got proof otherwise